airbyte-webapp/src/hooks/services/BulkEdit/BulkEditService.tsx

```diff
@@ -34,9 +34,15 @@ export const BulkEditServiceProvider: React.FC<
     update: (streams: SyncSchemaStream[]) => void;
   }>
 > = ({ children, nodes, update }) => {
+  const { setStatus } = useFormikContext();
   const [selectedBatchNodes, { reset, toggle, add }] = useSet<string | undefined>(new Set());
   const [options, setOptions] = useState<Partial<AirbyteStreamConfiguration>>(defaultOptions);
 
+  const isActive = selectedBatchNodes.size > 0;
+  useEffect(() => {
+    setStatus({ editControlsVisible: !isActive });
```

Comment:
To be safe, the current status should be merge in case it's ever expanded.

```ts
    setStatus({ ...status, editControlsVisible: !isActive });
```